boards/google/quincy/google_quincy_defconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 Google Inc
+# SPDX-License-Identifier: Apache-2.0
+
+# Serial Drivers
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# GPIO Controller
+CONFIG_GPIO=y
+
+# Clock Controller
+CONFIG_CLOCK_CONTROL=y
+
+# Enable MPU and HW stack protection
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
```

Comment:
I think this wants a `CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC` for some of the drivers to work, all other nuvoton boards have it, load of CI builds fail because it's unset https://github.com/zephyrproject-rtos/zephyr/pull/81548/checks?check_run_id=35000988928